package/gluon-mesh-babel/files/etc/hotplug.d/iface/90-gluon-babel-brclient-no-igmp-snooping

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+case "$ACTION"
+in
+	ifup)
+	echo 1 > /sys/devices/virtual/net/br-client/bridge/multicast_snooping
```

Comment:
File name seems to need an update, too. Also, multicast_snooping is enabled by default in the kernel (and OpenWRT+LEDE too), does it need to be set explicitly?

However, multicast_querier is not enabled by default (and I think the gluon-batman-adv packages do this?).